src/test/kotlin/ecommerce/controller/StatisticsE2ETest.kt

```diff
@@ -39,6 +43,9 @@ class StatisticsE2ETest {
     @Autowired
     private lateinit var cartItemRepository: CartItemRepository
 
+    @Autowired
+    private lateinit var optionRepository: OptionRepository
+  
```

Comment:
This might not seem like a big deal, but there are two whitespace characters in this file, in line 48.
Due to these whitespace characters, lint checking fails when I try to clean-build the project.
Why don't we remove these? :)